File: CompilerNetwork/ViewModels/ProblemTestCaseViewModel.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CompileNetwork.ViewModel
{
    public class ProblemTestCaseViewModel
    {
        public string ProblemTestCaseId { get; set; }
        public string ProblemId { get; set; }
        public string InputFilePath { get; set; }
        public string OutputFilePath { get; set; }
        public int? SizeInputFile { get; set; }
        public int? SizeOutputFile { get; set; }

        public IFormFile InputFile { get; set; }
        public IFormFile outFile { get; set; }
    }
}

File: CodeFreak1/ClientApp/src/app/problem/dtos/problem-test-case-view-model.ts
import { RequestStatus } from "../../Security/Dtos/request-status";

export class ProblemTestCaseViewModel extends RequestStatus {
  public ProblemTestCaseId: string;
  public ProblemId: string;
  public InputFilePath: string;
  public OutputFilePath: string;
  public SizeInputFile: number;
  public SizeOutputFile: number;
  public InputFile: File;
  public outFile: File;
}
